com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -523,6 +532,11 @@ public bool supportsTerrainHoles
             get { return m_SupportsTerrainHoles; }
         }
 
+        public StoreActionsOptimization storeActionsOptimization
+        {
+            get { return m_StoreActionsOptimization; }
+        }
```

Comment:
Needs public API. Is there any reason to not provide setter as well?

Reply:
we are only setting it in the pipeline settings, same as opaqueDownsampling or supportsTerrainHoles which don't have setters as well. So I just copied those. Since it is coming from pipeline settings seems safer to me to have it read only. wdyt?

Reply:
Some properties in the pipeline settings have setters, usually the ones that don't is because they affect keyword stripping so we don't allow users to change at runtime.

Reply:
thanks for the clarification, added the setter in the latest commit